examples/importers/csvbank.py

```diff
@@ -5,7 +5,7 @@
 
 
 class Importer(csvbase.Importer):
-    date = csvbase.Date('Posting Date', '%m/%d/%Y')
+    date = csvbase.Date('Posting Date', '%m/%d/%Y')  # type: ignore
```

Comment:
Why?

Reply:
This date column shadows the date function of the importer protocol. Seems like the base class initialisation makes that all work but I don't know if this can be typed correctly.